backend/app.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import cors from 'cors';
+
+const app = express();
+const axios= require("axios");
+
+app.use(cors({
+    origin: 'http://localhost:3000',
+    credentials: true
+}));
+
+app.get("/", (req, res) => {
+    res.send("Hello World!");
+});
+
+// TODO T32: sign up
+app.post('/signup', (req, res) => {
+
```

Comment:
let's add some dummy res.send's here as well so that these endpoints don't just timeout